app/javascript/components/supplemental_application/sections/RentalAssistance.js

```diff
@@ -165,85 +169,94 @@ const RentalAssistanceForm = ({ values, onSave, loading, onClose, applicationMem
-              className='rental-assistance-recipient'
-            />
-          </FormGrid.Item>
-        </FormGrid.Row>
-        {isOther(form.getState().values.rental_assistances[index]) && (
-          <FormGrid.Row expand={false}>
+        <InlineModal whiteBackground marginBottom={isNew} >
+          <FormGrid.Row>
+            <FormGrid.Item>
+              <SelectField
+                label='Recipient'
+                fieldName={`rental_assistances.${index}.recipient`}
+                options={applicationMembersOptions}
+                className='rental-assistance-recipient'
+              />
+            </FormGrid.Item>
+          </FormGrid.Row>
+          <FormGrid.Row>
             <FormGrid.Item>
-              <InputField
-                label='Other Assistance Name'
-                fieldName={`rental_assistances.${index}.other_assistance_name`} />
+              <SelectField
+                label='Type of Assistance'
+                fieldName={`rental_assistances.${index}.type_of_assistance`}
+                options={typeOfAssistanceOptions}
+                className='rental-assistance-type'
+                validation={validate.isPresent('Please select a type of assistance.')}
+              />
             </FormGrid.Item>
           </FormGrid.Row>
-        )}
-        <FormGrid.Row expand={false}>
-          <div className='form-grid_item column'>
-            <button
-              className='button primary tiny margin-right margin-bottom-none'
-              type='button'
-              onClick={onSaveWithIndex}
-              disabled={loading}>
-              Save
-            </button>
-            <button
-              className='button secondary tiny margin-right margin-bottom-none'
-              type='button'
-              onClick={onClose}
-              disabled={loading}>
-              Cancel
-            </button>
-            {!isNew && (
-              <button
-                className='button alert-fill tiny margin-bottom-none right'
-                type='button'
-                onClick={onDelete}
-                disabled={loading}>
-                Delete
-              </button>
-            )}
-          </div>
-        </FormGrid.Row>
+          <FormGrid.Row>
+            <FormGrid.Item>
+              <CurrencyField
+                label='Assistance Amount'
+                fieldName={`rental_assistances.${index}.assistance_amount`}
+                validation={validateAssistanceAmount}
+                id='assistance_amount'
+                isDirty={visited && visited[`rental_assistances.${index}.assistance_amount`]}
+              />
+            </FormGrid.Item>
+          </FormGrid.Row>
+          <FormGrid.Row>
+            <FormGrid.Item>
+              <YesNoRadioGroup
+                label='Recurring Assistance'
+                fieldName={`rental_assistances.${index}.recurring_assistance`}
+                uniqId={(!isEmpty(values) && values.id) || 'new'}
```

Comment:
could this be simplified to `values?.id || 'new'`?

Can you tell I'm excited that I just learned about the `?.` thing?

Reply:
OK So I googled it because you made me nervous that it's maybe not a thing

It looks like it "has not yet reached cross-browser stability" 😭  and until v80 of chrome even you had to enable it with a flag. Look at [implementation progress on this page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)

Not really sure what that means for us since everything is being minified anyways? Not sure if we have to test on browserstack or something

Reply:
Looks like we have `@babel/plugin-proposal-optional-chaining` in our `yarn.lock` so it should be compiled to something with all browsers.

We may want to explicitly add plugin-proposal-optional-chaining to our package.json file so that we know we'll always have it

Reply:
looks like it's ultimately coming from webpack so we should be all good. Thanks for checking!

as a bonus it looks like we get [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator]("nullish coalescing") too, which is like a null-aware ternary alternative. `const foo = null ?? 'default string';`  gives you 'default string'